src/design-system/components/private/ApplyShadow/ApplyShadow.tsx

```diff
@@ -85,7 +85,7 @@ function splitPositionStyles(style: ViewStyle) {
   ];
 }
 
-export const ApplyShadow = React.forwardRef(({ backgroundColor, children: child, shadows }: ApplyShadowProps, ref: React.Ref<any>) => {
+export const ApplyShadow = ({ backgroundColor, children: child, shadows }: ApplyShadowProps) => {
```

Comment:
There was some new warnings (prob because of the react upgrade) related to forwardRef ref param not being used. For this component if we are in the case `if (!shadows) return child;` the ref is not used. I noticed we actually never pass ref to this component so I just removed the forward ref.